tests/command_test.py

```diff
@@ -14,11 +14,23 @@
 
 
 class TestTest(unittest.TestCase):
+    def extract_json_data_from_bytes_array(self, input_bytes):
+        startIndex = 0
+        endIndex = 0
```

Comment:
```suggestion
        start_index = 0
        end_index = 0
```

nitpick: snake_case を使ってください。参考: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

(これほんとは pylint あたりで弾くべきなんですが、まだ設定が甘いです。ごめんね :bow:)